requirements.txt

```diff
@@ -196,6 +196,8 @@ pillow==11.1.0
     # via
     #   matplotlib
     #   randovania (setup.py)
+planets-yapr==1.0.0
```

Comment:
linter is failing because pythonnet needs to be updated to be refernced from planets now too

Reply:
fixed it